modules/tensor_mechanics/src/materials/ComputeLinearElasticStress_CUT_DT.C

```diff
@@ -0,0 +1,50 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "ComputeLinearElasticStress_CUT_DT.h"
+
+template<>
+InputParameters validParams<ComputeLinearElasticStress_CUT_DT>()
+{
+  InputParameters params = validParams<ComputeStressBase>();
+  params.addClassDescription("Compute stress using elasticity for small strains");
+  return params;
```

Comment:
Yes, please don't merge this ComputeLinearElasticStress_CUT_DT - it was just meant as an illustration of the "Impossible" error.  It's a completely useless Material otherwise.  I can provide a realistic and useful Material if you like, but it'll be in ComputeMultiPlasticityStress, which is way more complicated.